src/drivers/github.js

```diff
@@ -273,15 +273,19 @@ class Github {
       }
 
       await exec(
-        `${resolve(
-          workdir,
-          'config.sh'
-        )} --unattended --token "${await this.runnerToken()}" --url "${
-          this.repo
-        }" --name "${name}" --labels "${labels}" --work "${resolve(
-          workdir,
-          '_work'
-        )}" ${single ? ' --ephemeral' : ''}`
+        resolve(workdir, 'config.sh'),
+        '--unattended',
+        '--token',
+        await this.runnerToken(),
+        '--url',
+        this.repo,
+        '--name',
+        name,
+        '--labels',
+        labels,
+        '--work',
+        resolve(workdir, '_work'),
+        ...(single ? ['--ephemeral'] : [])
```

Comment:
please include a quick note for future readers/contributors

```suggestion
        // note: '...' unpacking on an empty array to achieve a no-op effect
        ...(single ? ['--ephemeral'] : [])
```

Reply:
```suggestion
        // adds `--ephemeral` to the array only if `single` is set
        ...(single && ['--ephemeral'])
```

Reply:
I would say its use for object unpacking is more common than for arrays. 

Your linked example is already barf js code, and I would not consider it an excuse to continue. Consider that in your linked code, this is the norm, but here in this new chunk, it is out of place. Thus a quick footnote would be nice.

I don't understand your allergic reaction to code comments.

side note I think that `['thing', ...(false && ['maybe thing']) ]` works

Reply:
> _Your linked example is already barf js code, and I would not consider it an excuse to continue._

Speaking of which, if there is any more idiomatic syntax, I'd be glad to change it. It's not my cup of tea either.

> _Consider that in your linked code, this is the norm, but here in this new chunk, it is out of place. Thus a quick footnote would be nice._

That's why I committed a comment, with the _footnote_. 🦶🏼🔫 

> _I don't understand your allergic reaction to code comments._

Sorry, it's probably the season. You know, pollen, dust, https://github.com/iterative/leo-server/pull/26#pullrequestreview-1204495281, ... 😄 

> _side note I **don't** think that `['thing', ...(false && ['maybe thing']) ]` works_

You're right, moved to https://github.com/iterative/cml/pull/1275#discussion_r1040339366